src/AppBundle/Entity/LegislativeCandidate.php

```diff
@@ -241,4 +251,14 @@ public function setEmailAddress(?string $emailAddress): void
     {
         $this->emailAddress = $emailAddress;
     }
+
+    public function getCareer(): ?string
+    {
+        return $this->career;
+    }
+
+    public function setCareer(?string $career)
```

Comment:
It's not nullable (based on the annotation) so you should remove the `?` or put it nullable.